test/testinput/bufr_ncep_atms_remap.yaml

```diff
@@ -179,4 +180,5 @@ observations:
           source: variables/remappedBT
           longName: "3-by-3 Averaged Brightness Temperature"
           units: "K"
+          range: [120, 500]
```

Comment:
@emilyhcliu this is the range in a prior ATMS converter the high value is a bit high, and the low is actually a bit close.

would you be OK with range: [80, 380]

or do you think this is a nothing-burger ;0)